llvm/lib/Target/NVPTX/NVPTXTargetTransformInfo.cpp

```diff
@@ -587,6 +588,9 @@ Value *NVPTXTTIImpl::rewriteIntrinsicWithAddressSpace(IntrinsicInst *II,
       return ConstantInt::get(II->getType(), *R);
     return nullptr;
   }
+  case Intrinsic::nvvm_prefetch_tensormap: {
+    return nullptr;
```

Comment:
In order for this to be useful we need to actually rewrite the intrinsic here. That is, create an `nvvm_prefetch_tensormap` intrinsic with NewV. It would also be good to add a couple tests to confirm this works as expected.